academic-services/graph/network-similarity-usql-getsimilarity.md

```diff
@@ -0,0 +1,52 @@
+---
+title: NetworkSimilarity.GetSimilarity Function
+description: NetworkSimilarity.GetSimilarity Function
+services: microsoft-academic-services
+ms.topic: extra
+ms.service: microsoft-academic-services
+ms.date: 9/23/2020
+---
+# NetworkSimilarity.GetSimilarity Function
+
+Namespace: AcademicGraph.NetworkSimilarity
+
+### Compute the similarity score between two entities.
+
+## GetSimilarity(BaseDir, EntityType, Sense, EntityId1, EntityId2)
```

Comment:
>BaseDir, EntityType, Sense, EntityId1, EntityId2 [](start = 17, length = 48)

nit: parameters are camel cased in other methods, but capitalized here.

Reply:
The Python methods parameters are camel cased. USQL functions parameters are capitalized.

---
In reply to: [501882795](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/183#discussion_r501882795) [](ancestors = 501882795)

Reply:
Ah, very good.  TIL :). 

---
In reply to: [501885528](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/183#discussion_r501885528) [](ancestors = 501885528,501882795)